src/mount/mount-tool.c

```diff
@@ -330,7 +330,17 @@ static int parse_argv(int argc, char *argv[]) {
                         return -EINVAL;
                 }
 
-                if (arg_transport == BUS_TRANSPORT_LOCAL) {
+                if (arg_mount_type
+                    && STR_IN_SET(arg_mount_type,
+                                  // virtual file system
+                                  "autofs", "cgroup", "cgroup2", "configfs", "debugfs", "devpts", "devtmpfs", "efivarfs",
+                                  "hugetlbfs", "mqueue", "proc", "pstore", "ramfs", "securityfs", "sysfs", "tmpfs",
+                                  // network file system
+                                  "cifs", "nfs", "nfs4", "smbfs")) {
```

Comment:
maybe externalize this set somewhere, so it's define at a single place and easy to update ?

or is that too forward-looking ?

Reply:
Uh, I am not sure I am overly enthusiastic about maintaining these lists…

 That said, we already have a function for the network tests in fstype_is_network(). And I figure adding fstype_is_api_vfs() would be acceptable...

Reply:
OK. I will update.